openshift/koku-template.yaml

```diff
@@ -98,7 +99,7 @@ objects:
         containers:
         - env:
           - name: DATABASE_SERVICE_NAME
-            value: POSTGRES_SQL
+            value: ${DATABASE_SERVICE_NAME}
```

Comment:
Don't change this value. DATABASE_SERVICE_NAME is an env variable used by the Django `environ` module. 

So, every other place that is using the templated value by the same name is fine to change. Just not this one. :sweat_smile: 

We may want to rename the openshift template parameter to a different variable name just to reduce confusion. :stuck_out_tongue_winking_eye: